lib/assessment.rb

```diff
@@ -13,28 +18,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce do |item, memo|
+      memo += item
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any?{|x| x.odd?}
```

Comment:
Correct! This could also have been written as 
`numbers.any?(&:odd?)` 
(or, in Ruby 2.0 and above, as just `numbers.any?(:odd?)`)